src/views/Documents.vue

```diff
@@ -80,124 +79,105 @@
-        this.$router.push(url);
-      },
       changeSectionNavContent: function (event) {
-        let version = this.getVersionString();
-        var chapter = event.currentTarget.className;
-        var section = event.currentTarget.innerText;
-        console.log(chapter);
-        this.updateHeader(version, chapter.trim(), section);
-        this.$route.params.section = "sec" + chapter.replace(/[^0-9]/ig, "");
-        this.fetchData();
-
-        location.href = "#" + section.toLocaleLowerCase().replace(/ /g, '-');
+        let chapter = event.currentTarget.className;
+        let section = event.currentTarget.innerText;
+        this.$route.params.chapter = "chap" + chapter.replace(/[^0-9]/ig, "");
+        this.$route.params.section = "sec" + section.replace(/[^0-9]/ig, "");
+        let url = "/Documents/" + this.getVersion() + "/" + this.getChapter() + "/" + this.getSection();
+        this.$router.push(url);
+        history.go(0);
       },
       // get the version
       getVersion() {
         return this.$route.params.version;
       },
+      // get the chapter
+      getChapter() {
+        return this.$route.params.chapter;
+      },
       // get the section
       getSection() {
         return this.$route.params.section;
       },
       updateDocument(){
-        this.updateVersion();
-        this.updateSection();
-      },
-      updateVersion() {
         this.text = this.getVersionString();
-        this.generateCatalogue();
-      },
-      updateSection() {
-        let version = this.getVersion();
-        let chapter = Number(this.getSection().substr(3)) - 1;
-        if(chapter >= 0 && chapter <  Golbal.SUPPORT_VERSION[version]['chapters'].length){
-          this.chapter = Golbal.SUPPORT_VERSION[version]['chapters'][chapter].substr(0, Golbal.SUPPORT_VERSION[version]['chapters'][chapter].length - 3);
-        } else {
-          this.$router.push('/404');
-        }
-      },
-      updateHeader(chapter, section, content){
-        this.chapter = section;
-        this.content = content;
       },
       // use version and section to render markdown
       fetchData() {
         let version = this.getVersion();
-        let chapter = Number(this.getSection().substr(3)) - 1;
-        let url = null;
-        if (version in Golbal.SUPPORT_VERSION) {
-          url = Golbal.SUPPORT_VERSION[version]['doc-prefix'] + Golbal.SUPPORT_VERSION[version]['branch'] +
-            "/docs/Documentation/UserGuide" + Golbal.SUPPORT_VERSION[version]['version'] + "/" +
-            Golbal.SUPPORT_VERSION[version]['chapters'][chapter];
+        if (version in Global.SUPPORT_VERSION) {
+          let chapter = Number(this.getChapter().substr(4)) - 1;
+          let section = Number(this.getSection().substr(3));
+          let url = 'https://raw.githubusercontent.com/samperson1997/incubator-iotdb/' + Global.SUPPORT_VERSION[version]['branch'] +
```

Comment:
use your own repo?

Reply:
I notice that...  After the PR of the other repository is merged, I’ll change it immediately